symplyphysics/definitions/wave_equation_in_one_dimension.py

```diff
@@ -0,0 +1,38 @@
+from sympy import (
+    Derivative,
+    Eq,
+    symbols,
+    Function as SymFunction,
+)
+from symplyphysics import (
+    units,
+    Symbol,
+    print_expression,
+)
+
+# Description
+## The wave equation is a second-order linear partial differential equation used to
+## describe the description of waves, including standing wave fields such as mechanical
+## or electromagnetic waves.
+
+# Law: d**2(u(x, t))/dx**2 = (1/v**2) * d**2(u(x, t))/dt**2
+## u(x, t) - factor representing a displacement from rest situation
```

Comment:
Actually it does not look one-dimensional case. It looks like a parametrized two-dimensional (plus time) trajectory.

Reply:
I meant 1 spatial dimension

Reply:
Yes, but it's about two spatial dimensions. One is argument of a function, and second is the function's output.

Reply:
![image](https://github.com/blackyblack/symplyphysics/assets/115647936/7c4f9149-9fcf-47f5-a8e6-0d635691e261)

Looking at the terminology it seems that they call it _one-dimensional_ since the function depends on one spatial dimension. What's more, the output of the function does not need to be another spatial dimension, it can be any physical quantity and maybe even the same spatial dimension.

Reply:
Thanks.
We could add it to description though.

Reply:
I will